Repository/Models/Genre.cs

```diff
@@ -7,7 +7,14 @@ namespace Repository.Models
 {
     public partial class Genre
```

Comment:
purpose

Reply:
That is a database model, created by the database-first scaffolding tool.